pipeline.py

```diff
@@ -72,31 +90,89 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        for article_item in self.corpus_manager.get_articles().values():
+            tokens = self._process(article_item.get_raw_text())
+            tokens_for_article = []
+            single_tagged_tokens = []
+            multiple_tagged_tokens = []
+            for token in tokens:
+                tokens_for_article.append(token.get_cleaned())
+                single_tagged_tokens.append(token.get_single_tagged())
+                multiple_tagged_tokens.append(token.get_multiple_tagged())
+            article_item.save_as(' '.join(tokens_for_article), kind='cleaned')
+            article_item.save_as(' '.join(single_tagged_tokens), kind='single_tagged')
+            article_item.save_as(' '.join(multiple_tagged_tokens), kind='multiple_tagged')
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+        article_text = raw_text.replace('-\n', '')
+        for symbol in ['\n', '\r']:
+            article_text = article_text.replace(symbol, ' ')
+        m_tokens_list = []
+        analyzed_text_mystem = Mystem().analyze(article_text)
+        pymorphy2_analyzer = pymorphy2.MorphAnalyzer()
+        for analyzed_word in analyzed_text_mystem:
+            if analyzed_word['text'] == ' ':
+                continue
+            original_word = analyzed_word['text']
+            if 'analysis' not in analyzed_word.keys():
+                continue
+            if not analyzed_word['analysis']:
+                continue
+            if 'lex' not in analyzed_word['analysis'][0].keys():
+                continue
+            if 'gr' not in analyzed_word['analysis'][0].keys():
+                continue
+            token = MorphologicalToken(original_word)
+            token.normalized_form = analyzed_word['analysis'][0].get('lex')
+            token.tags_mystem = analyzed_word['analysis'][0].get('gr')
+            m_tokens_list.append(token)
```

Comment:
a cleaner solution would be append only after you fill it, otherwise it is hard to follow